.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
how does this work? doesn't look like this has any actions

Reply:
True - we require a file though and this is what I was told to put here for now 😄